--- thickbox.js.orig	2009-11-03 15:36:58.310484745 +0200
+++ thickbox.js	2009-11-03 15:39:25.736303166 +0200
@@ -31,6 +31,27 @@
 function tb_show(caption, url, imageGroup) {//function called when the user clicks on a thickbox link
 
 	try {
+
+		// Hide some things, because they peek trough
+		$('select, iframe, object').css({ 'visibility' : 'hidden' });
+		// and still show our result flashes
+		var i, embeds = document.getElementsByTagName("embed");
+		for (i = 0; i != embeds.length; i++) {
+			if (embeds[i]['id'] != 'ampie') {
+				embeds[i].style.visibility = "hidden";
+			} else {
+				embeds[i].style.visibility = "";
+			}
+		}
+		embeds = document.getElementsByTagName("object");
+		for (i = 0; i != embeds.length; i++) {
+			if (embeds[i]['id'] != 'ampie') {
+				embeds[i].style.visibility = "hidden";
+			} else {
+				embeds[i].style.visibility = "";
+			}
+		}
+
 		if (!document.body.style.maxHeight && document.documentElement.clientHeight) {//if IE 6 with doctype
 			$("body","html").css({height: "100%", width: "100%"});
 			$("html").css("overflow","hidden");
@@ -276,6 +297,8 @@
 	}
 	document.onkeydown = "";
 	document.onkeyup = "";
+
+	$('select, iframe, object, embed').css({ 'visibility' : 'visible' });
 	return false;
 }
 
